### ISSUE TITLE
Release SSDLite MobileDet GPU pretrained model as mentioned in the Object Detection API README

### ISSUE THREAD

**Issue by louisquinn** [tensorflow/models#9132]
> MobileDet GPU
We have released SSDLite with MobileDet GPU backbone, which achieves 17% mAP higher than the MobileNetV2 SSDLite (27.5 mAP vs 23.5 mAP) on a NVIDIA Jetson Xavier at comparable latency (3.2ms vs 3.3ms).
Along with the model definition, we are also releasing model checkpoints trained on the COCO dataset.

Hi all, 

I can't seem to find the pre-trained SSDLite with MobileDet GPU model in both the TF1 and TF2 model zoos as referenced by the README. Are there plans to release it? 

Thanks :)

**Comment by skn047** [comment]
I have the same question.
I guess ssd_mobiledet_edgetpu_coco in TF1 model zoo is that you're looking for.
Is this right?

**Comment by louisquinn** [comment]
@skn047 
`ssd_mobiledet_edgetpu_coco` seems to be explicitly a "Pixel4 Edge TPU model" and its mAP differs from what is written in the README (27.5 vs 25.9/25.6).

It is built from `class SSDMobileDetEdgeTPUFeatureExtractor(SSDMobileDetFeatureExtractorBase)` here:
https://github.com/tensorflow/models/blob/master/research/object_detection/models/ssd_mobiledet_feature_extractor.py#L533

I'm asking about the trained model that would be built from `class SSDMobileDetGPUFeatureExtractor(SSDMobileDetFeatureExtractorBase)` here:
https://github.com/tensorflow/models/blob/master/research/object_detection/models/ssd_mobiledet_feature_extractor.py#L561

**Comment by rsandler00** [comment]
Yes, same question! The sample config file for it exists here: `https://github.com/tensorflow/models/blob/master/research/object_detection/samples/configs/ssdlite_mobiledet_gpu_320x320_coco_sync_4x4.config`, but there are no weights for it in the model zoo [here](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf1_detection_zoo.md)! (even though there are pretrained weighted for every other MobileDet variant including TPU, DSP, and CPU...)
